projects/angular/src/forms/datepicker/_datepicker.clarity.scss

```diff
@@ -445,81 +276,26 @@
   .year {
     @include generate-calendar-button($padding: datepicker-variables.$clr-calendar-month-cell-padding);
     @include generate-calendar-hover-active-color-style();
-    // this is already generated through calendar-btn class. Commented for now
-    //@include generate-calendar-focus-style();
-
-    @include mixins.css-var(
-      height,
-      clr-calendar-month-year-cell-height,
-      datepicker-variables.$clr-calendar-month-cell-height,
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(
-      width,
-      clr-calendar-month-year-cell-width,
-      datepicker-variables.$clr-month-year-btn-width,
-      variables.$clr-use-custom-properties
-    );
+    @include mixins.generate-typography-token('SECONDARY-13-RG-CPT');
 
-    //Other Props
-    outline-offset: -1 * variables.$clr_baselineRem_5px; //needed for chrome
+    height: datepicker-variables.$clr-calendar-month-cell-height;
+    width: datepicker-variables.$clr-month-year-btn-width;
 
-    @include mixins.css-var(
-      font-size,
-      clr-calendar-month-year-font-size,
-      datepicker-variables.$clr-calendar-picker-btn-font-size,
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(
-      font-weight,
-      clr-calendar-month-year-font-weight,
-      datepicker-variables.$clr-calendar-picker-btn-font-weight,
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(
-      line-height,
-      clr-calendar-month-year-line-height,
-      datepicker-variables.$clr-calendar-click-target,
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(
-      letter-spacing,
-      clr-calendar-month-year-letter-spacing,
-      inherit,
-      variables.$clr-use-custom-properties
-    );
     &.is-disabled {
-      //ATM: remove opacity mixin and color calc
-      @include mixins.css-var(opacity, clr-calendar-disable-opacity, 0.4, variables.$clr-use-custom-properties);
-      @include mixins.css-var(color, clr-calendar-disable-color, inherit, variables.$clr-use-custom-properties);
+      color: tokens.$cds-alias-object-interaction-color-disabled;
       pointer-events: none;
       &:hover {
         background: none;
       }
     }
 
     &.is-selected {
-      @include mixins.css-var(
-        background,
-        clr-calendar-active-cell-background-color,
-        datepicker-variables.$clr-calendar-active-cell-background-color,
-        variables.$clr-use-custom-properties
-      );
-      @include mixins.css-var(
-        color,
-        clr-calendar-active-cell-color,
-        datepicker-variables.$clr-calendar-active-cell-color,
-        variables.$clr-use-custom-properties
-      );
+      background: datepicker-variables.$clr-calendar-active-cell-background-color;
```

Comment:
I think we should be switching to `background-color` when we create/update content.
No need to mass replace, though it won't be too bad.

Reply:
Ok, if our property is called "color" then it would better be color.
But for backward compatibility sake, we'd better leave it open for any possible overrides with img backgrounds.
So, I agree to keep this as it is.